src/lib/Scenes/City/CityPicker.tsx

```diff
@@ -34,6 +34,7 @@ export class CityPicker extends Component<Props, State> {
 
   selectCity(city: string, index: number) {
     this.setState({ selectedCity: city })
+    // TODO: setState is asynchronous. Why are we doing this twice?
```

Comment:
I talked to @ashleyjelks and she explained why we're doing this. She's going to follow-up with a PR to use the callback on `setState`.